modules/ROOT/pages/clustering/databases.adoc

```diff
@@ -225,6 +225,109 @@ neo4j@neo4j> DRYRUN REALLOCATE DATABASES;
+See xref:clustering/disaster-recovery.adoc[] for more information.
+ 
+[CAUTION]
+====
+The recreate procedure works only for real user databases and not for composite databases, or the `system` database.
+
+Keep in mind the recreate procedure results in downtime while the stores get updated.
+The time is unbounded and may depend on the store size and network speed.
+
+The recreate procedure selects the most up-to-date seed from the defined seeding servers.
+If only a subset of the previous cluster members is specified as seeders, there is a risk of data loss.
+To avoid this, ensure you back up the current store before replacing it with a seed from a URI, so the latest state can be restored if needed.
+====
+
+The database in question can be in an `online` or `offline` state when it is recreated, but a successful operation starts the database regardless of the previous state.
+
+If CDC is enabled for the database, it will still be enabled after recreating the database, however the chain will be broken.
+See link:https://neo4j.com/docs/cdc/current/[CDC documentation] for more details.
+
+Before recreating a database, any eventual quarantined states need to be addressed.
+For more information, refer to the quarantined state documentation.
+
+You need xref:authentication-authorization/dbms-administration.adoc#access-control-dbms-administration-database-management[the DBMS `DATABASE MANAGEMENT` privileges] to run the recreate procedure. 
+
+[[recreate-seeding-options]]
+=== Seeding options and configurations
+
+The store to be used during the recreation can be defined in three different ways.
+One method uses a backup, while the two others use available allocations in the cluster.
+
+Additionally, you have the option to modify <<alter-topology-recreate, the topology>> during the recreation process.
+However, note that the store format, access, and enrichment cannot be altered during recreation.
+
+[[uri-seed]]
+==== `seedURI`
+
+If you provide a URI to a backup or a dump, the stores on all allocations will be replaced by the backup or the dump at the given URI.
+The new allocations can be put on any `ENABLED` server in the cluster.
+See <<cluster-seed-uri, Seed from URI>> for more details.
+
+[source, shell]
+----
+CALL dbms.cluster.recreateDatabase("neo4j", {seedURI: "myBackup"});
+----
+
+[[seed-servers]]
+==== `seedingServers`
+
+Specify a set of available servers.
+The stores on all allocations will be synchronized to the most up-to-date store from the defined servers.
+The number of defined servers cannot exceed the number of total allocations in the desired topology. 
+
+[source, shell]
+----
+CALL dbms.cluster.recreateDatabase("neo4j", {seedingServers: ["serverId1", "serverId2", "serverId3"]});
+----
+
+[[undefined-servers]]
+==== Undefined servers
+
+If you provide an empty list of seeding servers or do not specify a `seedURI`, Neo4j automatically selects all available allocations of the database as seeders.
```

Comment:
```suggestion
If you provide an empty list of seeding servers and do not specify a `seedURI`, Neo4j automatically selects all available allocations of the database as seeders.
```

Reply:
and

Either `seedURI` or `seedingServers` must be in the options. If you define neither that is an error.
If you define both then:
* either `seedingServers` is an empty list then see "Undefined servers with fallback backup"
* or `seedingServers` is not empty, that is an error

Reply:
The errors are being put in further up.